dynawo/sources/Models/Modelica/Dynawo/Examples/BaseClasses/GeneratorSynchronousInterfaces.mo

```diff
@@ -19,9 +19,6 @@ model GeneratorSynchronousInterfaces "Synchronous generator with real interfaces
 
   extends Dynawo.Electrical.Machines.OmegaRef.GeneratorSynchronous;
 
-  final parameter Dynawo.Types.CurrentModulePu Ir0Pu = IRotor0Pu;
```

Comment:
Why was it created in the first place ?

Reply:
IRororPu and IStatorPu were protected, thus inaccessible under OpenModelica.